cinema/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.CharField(max_length=255)
```

Comment:
Consider using `models.TextField()` for the `description` field instead of `models.CharField(max_length=255)`. This will allow for longer descriptions without the need to specify a maximum length.